sdk/storage/azfile/service/models.go

```diff
@@ -0,0 +1,101 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package service
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/internal/exported"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/internal/generated"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/share"
+)
+
+// SharedKeyCredential contains an account's name and its primary or secondary key.
+type SharedKeyCredential = exported.SharedKeyCredential
+
+// CreateShareOptions contains the optional parameters for the share.Client.Create method.
+type CreateShareOptions = share.CreateOptions
+
+// DeleteShareOptions contains the optional parameters for the share.Client.Delete method.
+type DeleteShareOptions = share.DeleteOptions
+
+// RestoreShareOptions contains the optional parameters for the share.Client.Restore method.
+type RestoreShareOptions = share.RestoreOptions
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+// GetPropertiesOptions provides set of options for Client.GetProperties
+type GetPropertiesOptions struct {
+	// placeholder for future options
+}
+
+func (o *GetPropertiesOptions) format() *generated.ServiceClientGetPropertiesOptions {
+	return nil
+}
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+// SetPropertiesOptions provides set of options for Client.SetProperties
+type SetPropertiesOptions struct {
+	// The set of CORS rules.
+	Cors []*ShareCorsRule
```

Comment:
Like blobs and queues, let's make all acronyms capitalized

Reply:
Done